app/src/main/res/values/strings.xml

```diff
@@ -300,6 +300,7 @@
   <string name="profile_rename_label">New Name</string>
   <string name="profile_rename_save">save</string>
   <!-- ProfileResetPinActivity -->
+  <string name="profile_reset_pin_activity_label">Reset PIN</string>
   <string name="profile_reset_pin_title">Reset PIN</string>
```

Comment:
Rename `profile_reset_pin_activity_label` to `profile_reset_pin_activity_title` and also remove `profile_reset_pin_title ` and use `profile_reset_pin_activity_label` whereever the string was getting used.